auth/role.go

```diff
@@ -12,13 +12,12 @@ package auth
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/couchbase/sync_gateway/base"
+	ch "github.com/couchbase/sync_gateway/channels"
 	"net/http"
 	"regexp"
 	"strconv"
 	"strings"
-
-	"github.com/couchbase/sync_gateway/base"
-	ch "github.com/couchbase/sync_gateway/channels"
 )
```

Comment:
This is super nit-picky, but I'm fairly sure our tooling (both Goland and `goimports`) should handle this by separating stdlib and third-party imports unless explicitly moved into the same group. Is there a reason why this and the `bcrypt` import in the file below changed?

Reply:
This was accidental — committed while removing the old import statement for sync package.